hudi-examples/hudi-examples-flink/pom.xml

```diff
@@ -263,6 +263,12 @@
             <scope>test</scope>
         </dependency>
         <!-- Junit 5 dependencies -->
+        <dependency>
```

Comment:
Why dependency changes are required in hudi-examples-flink and hudi-examples-spark?

Reply:
I would get the error:
`java.lang.NoClassDefFoundError: org/junit/platform/launcher/core/LauncherFactory`
both in my local environment and Azure CI if I did not add this dependency.

Reply:
got it